src/pages/feature/heehyun/Join/Join.js

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import './Join.scss';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import backArrow from './Back_arrow.png';
+
+const Join = () => {
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordVerify, setPasswordVerify] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [mobileNumber, setMobileNumber] = useState('');
+  const [profileImage, setProfileImage] = useState('');
+
+  const setChangeEmail = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const setChangePassword = (event) => {
+    setPassword(event.target.value);
+  };
+
+  const setChangePasswordVerify = (event) => {
+    setPasswordVerify(event.target.value);
+  };
+
+  const setChangeNickname = (event) => {
+    setNickname(event.target.value);
+  };
+
+  const setChangeMobileNumber = (event) => {
+    setMobileNumber(event.target.value);
+  };
+
+  const setChangeProfileImage = (event) => {
+    setProfileImage(event.target.value);
+  };
```

Comment:
선언된 state와  그 state들을 관리하기위한 비슷한 형태의 함수들이 굉장히 많습니다.
[계산된 속성명](https://velog.io/@ahn-sujin/React-input%EC%9D%B4-%EC%97%AC%EB%9F%AC%EA%B0%9C-%EC%9D%BC%EB%95%8C-%EC%83%81%ED%83%9C-%EA%B4%80%EB%A6%AC%ED%95%98%EA%B8%B0-%EA%B3%84%EC%82%B0%EB%90%9C-%EC%86%8D%EC%84%B1%EB%AA%85-%EC%82%AC%EC%9A%A9#:~:text=%EC%82%B4%ED%8E%B4%EB%B3%B4%EB%8F%84%EB%A1%9D%20%ED%95%98%EC%9E%90!-,%EA%B3%84%EC%82%B0%EB%90%9C%20%EC%86%8D%EC%84%B1%EB%AA%85%EC%9D%B4%EB%9E%80%3F,%EC%95%84%EB%8B%88%EB%9D%BC%20%5B%5D%20%EC%9D%84%20%EC%82%AC%EC%9A%A9%ED%95%9C%EB%8B%A4.)을 활용한다면 아래와 같이 축약 할 수 있습니다.
```suggestion
 const handleInputChange. =(e)=>{
 const { name, value } = e.target
 setState({...state,[name]:value})
 }
```